src/common/xio_session_client.c

```diff
@@ -871,6 +871,13 @@ static inline void xio_session_refuse_connection(void *conn)
 	xio_connection_refused(connection);
 }
 
+static void xio_connect_timeout(void *data)
+{
+        struct xio_connection *connection = (struct xio_connection *)data;
```

Comment:
Please fix spacing (tabs) according to conventions. Also on the next line of code.